### PR TITLE
fix some comment and log error

### PR DESCRIPTION
**Pull request by int88**
**What type of PR is this?**

> Uncomment one line below and remove others.
>
> Documentation

**What does this PR do? Why is it needed?**

fix some comment and log error

**Which issues(s) does this PR fix?**

Fixes #

**Other notes for review**

### CODE REVIEW

File: testing/endtoend/components/eth1/node_set.go [hunk 1]
```diff
@@ -52,7 +52,7 @@ func (s *NodeSet) Start(ctx context.Context) error {
 	})
 }
 
-// Started checks whether beacon node set is started and all nodes are ready to be queried.
+// Started checks whether Eth1 node set is started and all nodes are ready to be queried.
```
Comment (terencechain) [inline testing/endtoend/components/eth1/node_set.go:55]: nitpick

```suggestion
// Started checks whether execution node set is started and all nodes are ready to be queried.
```

**Comment by int88** [comment]
@terencechain updated.

File: testing/endtoend/component_handler_test.go [hunk 2]
```diff
@@ -110,7 +110,7 @@ func (c *componentHandler) setup() {
 	eth1Nodes := eth1.NewNodeSet()
 	g.Go(func() error {
 		if err := helpers.ComponentsStarted(ctx, []e2etypes.ComponentRunner{eth1Miner}); err != nil {
-			return errors.Wrap(err, "sending and mining deposits require ETH1 nodes to run")
+			return errors.Wrap(err, "ETH1 nodes require ETH1 miner to run")
```
Comment (terencechain) [inline testing/endtoend/component_handler_test.go:113]:
```suggestion
			return errors.Wrap(err, "Execution nodes require miner to run")
```

File: testing/endtoend/component_handler_test.go [hunk 3]
```diff
@@ -96,7 +96,7 @@ func (c *componentHandler) setup() {
 	eth1Miner := eth1.NewMiner()
 	g.Go(func() error {
 		if err := helpers.ComponentsStarted(ctx, []e2etypes.ComponentRunner{bootNode}); err != nil {
-			return errors.Wrap(err, "sending and mining deposits require ETH1 nodes to run")
+			return errors.Wrap(err, "ETH1 miner require boot node to run")
```
Comment (terencechain) [inline testing/endtoend/component_handler_test.go:99]:
```suggestion
			return errors.Wrap(err, "Miner require boot node to run")
```

File: testing/endtoend/component_handler_test.go [hunk 4]
```diff
@@ -110,7 +110,7 @@ func (c *componentHandler) setup() {
 	eth1Nodes := eth1.NewNodeSet()
 	g.Go(func() error {
 		if err := helpers.ComponentsStarted(ctx, []e2etypes.ComponentRunner{eth1Miner}); err != nil {
-			return errors.Wrap(err, "sending and mining deposits require ETH1 nodes to run")
+			return errors.Wrap(err, "Execution nodes require miner to run")
```
Comment (terencechain) [inline testing/endtoend/component_handler_test.go:113]:
```suggestion
			return errors.Wrap(err, "execution nodes require miner to run")
```

File: testing/endtoend/component_handler_test.go [hunk 5]
```diff
@@ -137,7 +137,7 @@ func (c *componentHandler) setup() {
 	proxies := eth1.NewProxySet()
 	g.Go(func() error {
 		if err := helpers.ComponentsStarted(ctx, []e2etypes.ComponentRunner{eth1Nodes}); err != nil {
-			return errors.Wrap(err, "beacon nodes require ETH1 and boot node to run")
+			return errors.Wrap(err, "proxies require ETH1 nodes to run")
```
Comment (terencechain) [inline testing/endtoend/component_handler_test.go:140]:
```suggestion
			return errors.Wrap(err, "proxies require execution nodes to run")
```

File: testing/endtoend/component_handler_test.go [hunk 6]
```diff
@@ -150,7 +150,7 @@ func (c *componentHandler) setup() {
 	beaconNodes := components.NewBeaconNodes(config)
 	g.Go(func() error {
 		if err := helpers.ComponentsStarted(ctx, []e2etypes.ComponentRunner{eth1Nodes, proxies, bootNode}); err != nil {
-			return errors.Wrap(err, "beacon nodes require ETH1 and boot node to run")
+			return errors.Wrap(err, "beacon nodes require proxies, ETH1 and boot node to run")
```
Comment (terencechain) [inline testing/endtoend/component_handler_test.go:153]:
```suggestion
			return errors.Wrap(err, "beacon nodes require proxies, execution and boot node to run")
```

File: testing/endtoend/component_handler_test.go [hunk 7]
```diff
@@ -164,7 +164,7 @@ func (c *componentHandler) setup() {
 		lighthouseNodes = components.NewLighthouseBeaconNodes(config)
 		g.Go(func() error {
 			if err := helpers.ComponentsStarted(ctx, []e2etypes.ComponentRunner{eth1Nodes, proxies, bootNode, beaconNodes}); err != nil {
-				return errors.Wrap(err, "lighthouse beacon nodes require ETH1 and boot node to run")
+				return errors.Wrap(err, "lighthouse beacon nodes require proxies, ETH1, beacon and boot node to run")
```
Comment (terencechain) [inline testing/endtoend/component_handler_test.go:167]:
```suggestion
				return errors.Wrap(err, "lighthouse beacon nodes require proxies, execution, beacon and boot node to run")
```

**Review by terencechain — APPROVED** [review]

**Pull request merged**